test/support/utils.js

```diff
@@ -0,0 +1,44 @@
+const stripAnsi = require("strip-ansi");
```

Comment:
Move this file to `test/helpers`. Rename it to `captureStdio` and expose only one function.